bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/service/LegalEntityService.kt

```diff
@@ -138,6 +143,21 @@ class LegalEntityService(
             externalId = externalId
         )
 
+    private fun toValidLegalEntities(legalEntityPage: Page<LegalEntity>): List<LegalEntityGateInputResponse> {
+
+        return legalEntityPage.content.map { legalEntity ->
```

Comment:
Is this method really needed to create a LegalEntityGateInputResponse object? I would expect a mapping method for LegalEntity to LegalEntityGateInputResponse in the ResponseMappings. Then just use this mapping directly and just delete toValidLegalEntities